tests/localosmosis/README.md

```diff
@@ -270,3 +270,9 @@ services:
 ```
 
 2. Checkout the Osmosis repository to a different `ref` that includes the new version, and then rebuild and restart LocalOsmosis using `make localnet-start`. Make sure to don't delete your `~/.osmosisd-local` folder.
+
+## FAQ
+
+Q: How do I enable pprof server in localosmosis?
+
+A: everything but the Dockerfile is already configured. Since we use a production Dockerfile in localosmosis, we don't want to expose the pprof server there by default. As a result, if you would like to use pprof, make sure to add `EXPOSE 6060` to the Dockerfile and rebuild the localosmosis image.
```

Comment:
What do you think about addition this to the production file but commenting it out so its even simpler to activate? Just a suggestion, not at all a blocker.

Reply:
Done